### PR TITLE
deploy recent updates to production

### PR DESCRIPTION
**Pull request by nibinqtl**

### CODE REVIEW

File: en/docs/edge-logic/built-in-variables.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ If you use the CDN Pro portal to edit the property, the autocompletion feature w
 | <span id="request_method"></span><span class="var">$request_method</span> | HTTP method: GET, POST, etc. | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
 | <span id="request_scheme"></span><span class="var">$request_scheme</span> | "http" or "https" | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
 | <span id="request_time"></span><span class="var">$request_time</span> |<span class="badge small" title="numerical value">#</span> request processing time in seconds with milliseconds resolution; time elapsed since the first byte was read from the client | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
-| <span id="request_uri"></span><span class="var">$request_uri</span> | request URI with query string received from client | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
+| <span id="request_uri"></span><span class="var">$request_uri</span> | request URI from the client, starting from '/' and with query string | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
```
Comment (burtonquantil) [inline en/docs/edge-logic/built-in-variables.md:55]: I suggest "request URI from the client, beginning with '/' and including the query string"

File: en/docs/edge-logic/built-in-variables.md [hunk 2]
```diff
@@ -84,4 +84,4 @@ If you use the CDN Pro portal to edit the property, the autocompletion feature w
...
 | <span id="upstream_trailer_"></span><span class="var">$upstream\_trailer\_<em>name</em></span> | trailer with the specified name passed from the Edge Logic with the [add_trailer](/docs/edge-logic/supported-directives#add_trailer) directive. | <span class="badge yellow">No</span> | <span class="badge green">Yes</span> |
-| <span id="uri"></span><span class="var">$uri</span> | [normalized](http://nginx.org/en/docs/http/ngx_http_core_module.html#location) request uri without query string. In Edge Logic, it may be modified by the "[rewrite](/docs/edge-logic/supported-directives#rewrite)" directive | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
+| <span id="uri"></span><span class="var">$uri</span> | [normalized](http://nginx.org/en/docs/http/ngx_http_core_module.html#location) request uri starting from '/' and without query string. The value may be modified by the "[rewrite](/docs/edge-logic/supported-directives#rewrite)" directive in the Edge Logic. | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
```
Comment (burtonquantil) [inline en/docs/edge-logic/built-in-variables.md:87]: I suggest  "request URI beginning with '/' and excluding the query string. "

File: en/docs/edge-logic/supported-directives.md [hunk 3]
```diff
@@ -621,9 +624,9 @@ Sets a timeout for `proxy_cache_lock`. If a request has been locked for this amo
 **Default:** `-` <br/>
 **Context:** server, location
 
-This directive has the same functionality as parameters `stale-if-error` and `stale-while-revalidate` in the `Cache-Control` header field. Its priority is lower than the header value.
+This directive allows the edge to serve staled content if it has not expired for too long to give the end user a better experience. It has the same functionality as the parameters `stale-if-error` and `stale-while-revalidate` in the `Cache-Control` header field. Its priority is lower than the header value.
```
Comment (burtonquantil) [inline en/docs/edge-logic/supported-directives.md:627]: Suggestion: This directive allows serving of stale content that did not expire too long ago to give the end user a better experience."

**Review by burtonquantil — CHANGES_REQUESTED** [review]
I made a few suggestions.

I'll upload the updated CDN Pro OpenAPI spec on Monday to reflect our most recent deployment's changes.

**Comment by nibinqtl** [comment]
updated with Burton's suggestions

File: en/docs/edge-logic/built-in-variables.md [hunk 4]
```diff
@@ -52,7 +52,7 @@ If you use the CDN Pro portal to edit the property, the autocompletion feature w
 | <span id="request_method"></span><span class="var">$request_method</span> | HTTP method: GET, POST, etc. | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
 | <span id="request_scheme"></span><span class="var">$request_scheme</span> | "http" or "https" | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
 | <span id="request_time"></span><span class="var">$request_time</span> |<span class="badge small" title="numerical value">#</span> request processing time in seconds with milliseconds resolution; time elapsed since the first byte was read from the client | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
-| <span id="request_uri"></span><span class="var">$request_uri</span> | request URI with query string received from client | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
+| <span id="request_uri"></span><span class="var">$request_uri</span> | request URI from the client, beginning with '/' and including query string | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
```
Comment (burtonquantil) [inline en/docs/edge-logic/built-in-variables.md:55]: I'd add "the" before "query string"

File: en/docs/edge-logic/built-in-variables.md [hunk 5]
```diff
@@ -84,4 +84,4 @@ If you use the CDN Pro portal to edit the property, the autocompletion feature w
...
 | <span id="upstream_trailer_"></span><span class="var">$upstream\_trailer\_<em>name</em></span> | trailer with the specified name passed from the Edge Logic with the [add_trailer](/docs/edge-logic/supported-directives#add_trailer) directive. | <span class="badge yellow">No</span> | <span class="badge green">Yes</span> |
-| <span id="uri"></span><span class="var">$uri</span> | [normalized](http://nginx.org/en/docs/http/ngx_http_core_module.html#location) request uri without query string. In Edge Logic, it may be modified by the "[rewrite](/docs/edge-logic/supported-directives#rewrite)" directive | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
+| <span id="uri"></span><span class="var">$uri</span> | [normalized](http://nginx.org/en/docs/http/ngx_http_core_module.html#location) request uri beginning with '/' and excluding the query string. The value may be modified by the "[rewrite](/docs/edge-logic/supported-directives#rewrite)" directive in the Edge Logic. | <span class="badge green">Yes</span> | <span class="badge green">Yes</span> |
```
Comment (burtonquantil) [inline en/docs/edge-logic/built-in-variables.md:87]: "uri" should be capitalized "URI"

**Review by burtonquantil — CHANGES_REQUESTED** [review]
couple minor things.

File: en/docs/edge-logic/built-in-variables.md (hunk 5 above)
Comment (nibinqtl) [reply en/docs/edge-logic/built-in-variables.md]: fixed.

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Comment by nibinqtl** [comment]
It seems the production site is not showing the latest content?

**Pull request merged**